packages/synthetix-main/contracts/submodules/PoolEventAndErrors.sol

```diff
@@ -30,5 +30,4 @@ contract PoolEventAndErrors {
     event RewardsClaimed(uint indexed poolId, address indexed token, uint indexed accountId, uint index, uint amountClaimed);
```

Comment:
This file is weird. Perhaps we should move everything out of it and delete the file.